### PR TITLE
Add specs for User#avatar_format

### PR DESCRIPTION
**Pull request by MrSerth**
While working on #1316, I got a few more tests suggested that I didn't want to remove (since none were present before).

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -208,4 +208,48 @@
       end
     end
   end
+
+  describe '#avatar_format' do
```
Comment (kkoehn) [inline spec/models/user_spec.rb:212]: I wouldn't add these tests as tests for a private method, but in the `#valid?` block.

**Review by kkoehn — CHANGES_REQUESTED** [review]
Great addition for the user-validations!
I added one small request concerning the locations of the specs.

File: spec/models/user_spec.rb (hunk 1 above)
Comment (MrSerth) [reply spec/models/user_spec.rb:212]: Done. ✅

**Review by Dome-GER — APPROVED** [review]

**Review by kkoehn — APPROVED** [review]
